validphys2/src/validphys/closuretest/multiclosure.py

```diff
@@ -148,6 +149,42 @@ def fits_dataset_bias_variance(
     return biases, np.asarray(variances), len(law_th)
 
 
+@check_multifit_replicas
+def fits_normed_dataset_central_delta(
+    internal_multiclosure_dataset_loader, _internal_max_reps=None, _internal_min_reps=20
+):
+    """
+    For each fit calculate the difference between central expectation value and true val. Normalize this
+    value by the variance of the differences between replicas and central expectation value (different
+    for each fit but expected to vary only a little). Each observable central exp value is
+    expected to be gaussianly distributed around the true value set by the fakepdf
```

Comment:
Please add also the return variable to this docstr, smt like 

```
deltas: np.array
    array of shape (n_fits, nreps)
```
(or whatever the actual shape is)


If you add the input arguments as in the docstr of the function  below even better.